MathlibTest/CalcQuestionMark.lean

```diff
@@ -0,0 +1,24 @@
+import Mathlib.Tactic.Widget.Calc
+
+/--
+info: Create calc tactic:
+• calc
+    1 = 1 := by sorry
+---
+warning: declaration uses 'sorry'
+-/
+#guard_msgs in
+example : 1 = 1 := by
+  calc?
```

Comment:
```suggestion
  have := 0
  calc?
```
so as to test also that `mdata` in the target does not affect the `calc?` tactic.